### DISCUSSION TITLE
Projects that use Clean Architecture?

### DISCUSSION THREAD

**Discussion by KenoLendzion** [jasontaylordev/CleanArchitecture#648] [General]
Hey! Does anyone know public GitHub repositories that use Clean Architecture? Would love to have a second or third reference while learning and trying to implement it.  

Thanks in advance!

**Comment by jasontaylordev** [comment]
I like the following repos:

- https://github.com/nopSolutions/nopCommerce - Onion Architecture
- https://github.com/dotnet-architecture/eShopOnWeb - Clean Architecture
- https://github.com/ardalis/CleanArchitecture - Ardalis Clean Architecture Template

It's always great to see different approaches and ideas.
